ipatests/test_integration/test_commands.py

```diff
@@ -1561,6 +1561,39 @@ def test_ipa_cacert_manage_prune(self):
 
         assert isrgrootx1_nick in result
 
+    def test_ipa_systemd_journal(self):
+        """
+        This testcase checks that administrative user credentials
+        is not leaked to journald log
+        """
+        server_cmds = (
+            ['ipa-server-install',
+             'ipa-adtrust-install',
+             'ipa-kra-install',
+             'ipa-restore',
+             'ipa-replica-manage',
+             'ipa-cs-replica-manage',
+             'ipa-server-certinstall',
+             ]
+        )
+        replica_cmds = (
+            ['ipa-replica-install',
+             ]
+        )
+        for cmd in server_cmds:
+            self.master.run_command(
+                [f'journalctl -t /usr/sbin/{server_cmds} -n1 -o json-pretty']
+            )
+            assert self.master.config.admin_password not in cmd.stdout_text
+            assert self.master.config.dirman_password not in cmd.stdout_text
+
+        for cmd in replica_cmds:
+            self.master.run_command(
+                [f'journalctl -t /usr/sbin/{replica_cmds} -n1 -o json-pretty']
+            )
```

Comment:
```
for cmd in replica_cmds:
            self.replicas[0].run_command(
                [f'journalctl -t /usr/sbin/{cmd} -n1 -o json-pretty']
            )
```

as you are validating logs for ipa-replica-install command which will run on replica.

Reply:
also if you are only validating single command, you will not need loop for it.